app/jobs/cache/browser_channels/responses_for_prefix.rb

```diff
@@ -95,6 +96,10 @@ def cleanup!
   rescue Errno::ENOENT
   end
 
+  def refresh_cdn!
+    Faraday.get(ENV["BRAVE_PCDN_URL"], {}, { "Cache-Control" => "no-cache"})
```

Comment:
This will only clear the cache for whatever edge node it happens to hit. Presumably the CDN has a way to actually invalidate the entire cache

Reply:
Going to revert this and take it up with devops to see what we can do for edge node invalidation.